cmd/utils.go

```diff
@@ -43,50 +43,20 @@ type DependencyOverview struct {
 }
 
 func getDepInfo(mainModules []string) *DependencyOverview {
-	depGraph := DependencyOverview{MainModules: mainModules}
 	// get output of "go mod graph" in a string
 	goModGraph := exec.Command("go", "mod", "graph")
+	fmt.Println(mainModules)
```

Comment:
Please remove this :)

Reply:
Oops forgot to remove the debug statements

Reply:
Fixed